src/components/TextInput/TextInput.stories.mdx

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+
+import TextInput from "./TextInput";
+import { TextInputTypes } from "./TextInputTypes";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+// eslint-disable-next-line id-blacklist
+import { text, boolean, select, number } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+
+export default {
+  title: "Form Elements/Text Input",
+  component: TextInput,
+};
```

Comment:
This export might not be necessary in MDX--I believe that is all handled by the Meta below.  Would recommend having a quick look at radio component to compare.